extensions/documents/homie_legacy _stats_extension.md

```diff
@@ -0,0 +1,67 @@
+# Legacy Stats
```

Comment:
I think there is still a whitespace in the name, apart from that all maintainers have approved and we are ready to merge